documentation/Installation.md

```diff
@@ -1508,6 +1511,23 @@ services:
     cgroupDriver: systemd
 ```
 
+#### kubeadm_kube-proxy
+
+*Installation task*: `deploy.kubernetes`
+
+*Can cause reboot*: No
+
+*Can restart service*: Always yes, `kubelet`
+
+*OS specific*: No
+
+In the `services.kubeadm_kube-proxy` section, you can override the original settings for kube-proxy.
+For more information about these settings, refer to the official Kubernetes documentation at [https://kubernetes.io/docs/reference/config-api/kube-proxy-config.v1alpha1/#kubeproxy-config-k8s-io-v1alpha1-KubeProxyConfiguration](https://kubernetes.io/docs/reference/config-api/kube-proxy-config.v1alpha1/#kubeproxy-config-k8s-io-v1alpha1-KubeProxyConfiguration).
+By default, the installer uses the following parameters:
+
+|Parameter|Default Value|
```

Comment:
There are no default parameters

Reply:
Yes, we do not offer default parameters for now, but they will be added later #573

Reply:
will be used for KubeProxyConntrackConfiguration